onnx/defs/tensor/defs.cc

```diff
@@ -780,3 +818,49 @@ For example A = [[1, 2], [3, 4]], B = [1, 2], tile(A, B) = [[1, 2, 1, 2], [3, 4,
         "T1",
         {"tensor(int64)"},
         "Constrain repeat's type to int64 tensors.");
+
+ONNX_OPERATOR_SCHEMA(Upsample)
```

Comment:
This is the only change in this file (adding upsample op). All others are format changes.

Reply:
@linkerzhang In case you are not aware of it: there is a `git-clang-format` script/tool that integrates with git to only clang-format the changed parts.

Reply:
thank you! I was hoping to keep doing this whole file formatting until whole code base follows the format :). I

Reply:
as this file is not that complex, so I'll keep the format changes for other parts.